### PR TITLE
refactor: upgrade to solidity 0.6.12, use new standard json file

### PR DESCRIPTION
**Pull request by hieronx**

### CODE REVIEW

File: bin/lender/deploy.sh [hunk 1]
```diff
@@ -100,6 +100,7 @@ addValuesToFile $DEPLOYMENT_FILE <<EOF
     "OPERATOR_FAB"       :  "$OPERATOR_FAB",
     "ASSESSOR_FAB"       :  "$ASSESSOR_FAB",
     "ASSESSOR_ADMIN_FAB" :  "$ASSESSOR_ADMIN_FAB",
+    "RESTRICTEDTOKEN_FAB" :  "$RESTRICTEDTOKEN_FAB",
```
Comment (xmxanuel) [inline bin/lender/deploy.sh]: maybe
```suggestion
    "RESTRICTED_TOKEN_FAB" :  "$RESTRICTEDTOKEN_FAB",
```

File: bin/util/util.sh [hunk 2]
```diff
@@ -55,27 +55,27 @@ addValuesToFile() {
...
-    echo 0x$(cat $DAPP_JSON | jq -r ".contracts[\"$1\"].bin")
+    echo 0x$(cat $DAPP_JSON | jq -r ".contracts[\"$1\"][\"$2\"].evm.bytecode.object")
 }
 
 
 getFabContract() {
-    CONTRACT_CODE=$(getContractCode $1)
-    SALT=$(seth --to-bytes32 $(seth --from-ascii $2))
-    FAB_ADDR="${!2}"
+    CONTRACT_CODE=$(getContractCode $1 $2)
```
Comment (xmxanuel) [inline bin/util/util.sh:63]: 👍

File: bin/util/util.sh [hunk 3]
```diff
@@ -55,27 +55,27 @@ addValuesToFile() {
...
+    SALT=$(seth --to-bytes32 $(seth --from-ascii $3))
+    FAB_ADDR="${!3}"
```
Comment (xmxanuel) [inline bin/util/util.sh:66]:
```suggestion
   # checks if a env variable with name of the fab is defined
    FAB_ADDR="${!3}"
```

**Review by xmxanuel — APPROVED** [review]

File: bin/lender/deploy.sh (hunk 1 above)
Comment (hieronx) [reply bin/lender/deploy.sh]: Makes sense 👍

**Pull request merged**